plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/business/api/image/ImageSelectionDialog.java

```diff
@@ -130,8 +132,10 @@ public class ImageSelectionDialog extends Dialog {
      *            if true there is only the close button instead of OK/Cancel
      * @param currentImagePath
      *            the path to the image as it is serialized in the model
+     * @param displayImagePaths
+     *            if true, display the current and new selected image path and a message if the image is not found.
      */
-    public ImageSelectionDialog(Shell parentShell, EObject contextObject, boolean onlyCloseButton, String currentImagePath) {
+    public ImageSelectionDialog(Shell parentShell, EObject contextObject, boolean onlyCloseButton, String currentImagePath, boolean displayImagePaths) {
```

Comment:
Sirius collab doesn't compile because of this modification. Update the patchset in Sirius collab.